schema/defs-image.json

```diff
@@ -15,22 +15,12 @@
       "id": "https://opencontainers.org/schema/image/manifestDescriptor",
       "type": "object",
       "required": [
-        "mediaType",
-        "size",
-        "digest",
+        "config",
         "platform"
       ],
       "properties": {
-        "mediaType": {
-          "description": "the mediatype of the referenced object",
-          "$ref": "#definitions/mediaType"
-        },
-        "size": {
-          "description": "the size in bytes of the referenced object",
-          "$ref": "defs.json#/definitions/int64"
-        },
-        "digest": {
-          "$ref": "#definitions/digest"
+        "config": {
+          "$ref": "#content-descriptor.json"
```

Comment:
Perhaps I naMaybe I'm a little problem on the naming or understand， but I think in the manifest_list.go, Descriptor is the need for specific, or else when the call will be given the wrong, like“Descriptor has no field or method validate”